doc/whats_new/v0.23.rst

```diff
@@ -127,3 +127,12 @@ Changelog
 
 - |Enhancement| improve error message in :func:`utils.validation.column_or_1d`.
   :pr:`15926` by :user:`Loïc Estève <lesteve>`.
+  
+:mod:`sklearn.feature_selection`
+.................................
+
+- |Enhancement| In :class:`feature_selection.RFE`  and
+  :class:`feature_selection.RFECV` added support for multioutput by setting
+  the argument multi_output=TRUE.
```

Comment:
Let's write,
```
- |Enhancement| Added support for multioutput  input in :class:`feature_selection.RFE`  and
  :class:`feature_selection.RFECV`.
```

Reply:
> 
> 
> One last comment below, also you need to resolve conflicts (can be done in Github UI).

Sure! I have resolved all the conflicts @rth
Sure! I have resolved all the conflicts and changed the .rst file @rth